src/QMCDrivers/DMC/DMCUpdatePbyPFast.cpp

```diff
@@ -192,22 +189,27 @@ void DMCUpdatePbyPWithRejectionFast::advanceWalker(Walker_t& thisWalker, bool re
     if(ibar)
     {
       int iat=nonLocalOps.id(ibar);
-      if(!W.makeMoveAndCheck(iat,nonLocalOps.delta(ibar))) return;
-      myTimers[2]->start();
-      Psi.acceptTMove(W,iat,w_buffer);
-#if 0
-      RealType ratio=Psi.ratio(W,iat,dG,dL);
-      W.G += dG;
-      W.L += dL;
-      //thisWalker.R[iat]=W.R[iat];
-      //w_buffer.rewind();
-      //W.copyToBuffer(w_buffer);
-      RealType logpsi = Psi.evaluateLog(W,w_buffer);
+#ifdef ENABLE_SOA
```

Comment:
RMC doesn't have Tmove enabled. The UseTMove was disabled by @rcclay in 2014.
pbyp and allp moves of AoS and SoA builds are both checked against v3.1.1. See my comment on #229